dashboard/db/migrate/20180403024415_add_primary_email_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddPrimaryEmailToUser < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :users, :primary_email, index: false
```

Comment:
I decided not to create a foreign key constraint here because I get really annoyed by them when doing database operations and I don't feel like they add much value, but I'm open to disagreement...